keyboards/k320/config.h

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2020 kuenhlee and Don Kjer
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+#define TKL_DURGOD_K320
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x2f68
+#define PRODUCT_ID      0xb381
```

Comment:
Even though this may be the OEM Vendor ID, that actually may be problematic. 

Also;
```suggestion
#define VENDOR_ID       0x2F68
#define PRODUCT_ID      0xB381
```

Reply:
Switched to Vendor ID `0xFEED` and Product ID `0x0320`